### PR TITLE
Fix links in documentation

### LINKED ISSUE #3674: [Documentation] Link broken
### To Reproduce
1. Visit [GREL-supported regex](https://docs.openrefine.org/manual/expressions/#grel-supported-regex) section of the OpenRefine documentation.
2. Select any link from the list (`The GREL functions that support regex are:`).
Notice the HTTP 404 Page not found errors for all of these links.

### Current Results
HTTP 404 Page not found when following any of these links.

### Expected Behavior
Correct page or section in documentation when following any of these links.

### PR DESCRIPTION
**Pull request by pvcodes**
Fixes #3674 

Changes proposed in this pull request:
- Links were sort of broken, but not that much

### CODE REVIEW

**Review by ostephens — CHANGES_REQUESTED** [review]
Thanks. Unfortunately these aren't quite the changes that are needed to address #3674 . Please see the comment from @joanneong in that issue to understand the corrections that need to be made

File: docs/docs/manual/expressions.md [hunk 1]
```diff
@@ -92,12 +92,12 @@ Do not use slashes to wrap regular expressions outside of a GREL expression.
 
 On the [GREL functions](grelfunctions) page, functions that support regex will indicate that with a “p” for “pattern.” The GREL functions that support regex are:
 *   [contains](grelfunctions#containss-sub-or-p)
-*   [replace](grelfunctions#replaces-s-or-p-find-s-replace)
-*   [find](grelfunctions#finds-sub-or-p)
+*   [replace](grelfunctions#find-and-replace)
+*   [find](grelfunctions#find-and-replace)
```
[the remark this answers is not in the document]
Comment (pvcodes) [reply docs/docs/manual/expressions.md:96]: Got it!

**Comment by wetneb** [comment]
Hi @pvcodes and @ostephens, it seems to me that this issue was actually resolved by #3821 already. Sorry that I did not find this issue and close it when I did that! So I would say this PR should only be used to fix the anchor parts of links (which can still be broken because that isn't validated by the CI at the moment unfortunately).

**Comment by wetneb** [comment]
Closing in favour of #3920

**Pull request closed without merging**